dex/src/main/scala/com/wavesplatform/dex/market/AggregatedOrderBookActor.scala

```diff
@@ -27,11 +28,12 @@ object AggregatedOrderBookActor {
     case class GetAggregatedSnapshot(client: ActorRef[OrderBookAggregatedSnapshot])              extends Query
   }
 
-  sealed trait Command extends Message
-  object Command {
-    case class ApplyChanges(levelChanges: LevelAmounts, lastTrade: Option[LastTrade], ts: Long) extends Command
-    case class AddWsSubscription(client: ActorRef[WsOrderBook])                                 extends Command
-    private[AggregatedOrderBookActor] case object SendWsUpdates                                 extends Command
+  sealed trait WsCommand extends Message
+  object WsCommand {
+    case class ApplyChanges(levelChanges: LevelAmounts, lastTrade: Option[LastTrade], ts: Long) extends WsCommand
```

Comment:
1. Actually, it is not a `WsCommand`. This is a regular order book update. An aggregated order book is used on REST API too.
2. Also see no reason to introduce a special `WsCommand` (Occam's razor)

Reply:
Done